trabalho-2/docker-compose.yaml

```diff
@@ -1,21 +1,12 @@
+version: '3.8'
+
```

Comment:
Esse atributo `version` foi deprecado há um tempo, o recomendado é não especificar version no docker-compose.yaml. Caso contrário a gente toma uns warnings nas versões mais recentes do docker.

https://github.com/compose-spec/compose-spec/blob/main/04-version-and-name.md